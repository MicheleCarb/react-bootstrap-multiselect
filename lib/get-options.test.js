import getOptions from './get-options';
test('get-options', () => {
  expect(getOptions()).toMatchSnapshot();
});
