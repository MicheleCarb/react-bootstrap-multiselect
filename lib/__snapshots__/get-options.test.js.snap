// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`get-options 1`] = `
Array [
  "allSelectedText",
  "buttonClass",
  "buttonContainer",
  "buttonText",
  "buttonTitle",
  "buttonWidth",
  "checkboxName",
  "delimiterText",
  "disableIfEmpty",
  "disabledText",
  "dropRight",
  "dropUp",
  "enableCaseInsensitiveFiltering",
  "enableClickableOptGroups",
  "enableCollapsibleOptGroups",
  "enableFiltering",
  "enableFullValueFiltering",
  "enableHTML",
  "filterBehavior",
  "filterPlaceholder",
  "includeFilterClearBtn",
  "includeSelectAllIfMoreThan",
  "includeSelectAllOption",
  "inheritClass",
  "maxHeight",
  "nSelectedText",
  "nonSelectedText",
  "numberDisplayed",
  "onChange",
  "onDeselectAll",
  "onDropdownHidden",
  "onDropdownHide",
  "onDropdownShow",
  "onDropdownShown",
  "onFiltering",
  "onInitialized",
  "onSelectAll",
  "optionClass",
  "optionLabel",
  "preventInputChangeEvent",
  "selectAllJustVisible",
  "selectAllName",
  "selectAllNumber",
  "selectAllText",
  "selectAllValue",
  "selectedClass",
  "templates",
]
`;
